### PR TITLE
add module "onion"

### PR DESCRIPTION
**Pull request by runnerSnail**
https://github.com/runnerSnail/onion

### CODE REVIEW

**Comment by kevinkassimo** [comment]
Hi, could you explain what has been changed in this PR, thanks

**Comment by runnerSnail** [comment]
Sorry I added a link to my repository 
> [onion](https://github.com/runnerSnail/onion) - Web middleware that relies on deno_std http package

**Comment by zekth** [comment]
https://github.com/runnerSnail/onion => This repository is empty.

**Comment by runnerSnail** [comment]
> https://github.com/runnerSnail/onion => This repository is empty.

sad,I'm sorry, but it's all right now

File: README.md [hunk 1]
```diff
@@ -2,143 +2,144 @@
...
-- [Tools](#tools)
-- [Articles](#articles)
-- [Presentations](#presentations)
-- [Newsletters](#newsletters)
-- [Resources in Other Languages](#resources-in-other-languages)
-  - [Chinese](#chinese)
-  - [Hebrew](#hebrew)
-  - [Japanese](#japanese)
-  - [Korean](#korean)
+-   [Docs](#docs)
```
Comment (kevinkassimo) [inline README.md]: Could you keep the original formatting of entries?

Comment (runnerSnail) [reply README.md]: ok,done

**Comment by kevinkassimo** [comment]
Also could you slightly update the README file in your repo? It seems show the name `http-compose` instead and probably also needs a bit more description (see https://github.com/oakserver/oak as an example)

**Pull request closed without merging**